### PR TITLE
only store one old backup and one current backup during the update script to limit the used storage

### PR DESCRIPTION
**Pull request by szaimen**
Fix https://github.com/nextcloud/vm/issues/1942
See my comment in https://github.com/nextcloud/vm/issues/1942#issuecomment-834812405

Signed-off-by: szaimen <szaimen@e.mail.de>

### CODE REVIEW

File: nextcloud_update.sh [hunk 1]
```diff
@@ -732,12 +732,12 @@ fi
...
 print_text_in_color "$ICyan" "Backing up data..."
-DATE=$(date +%Y-%m-%d-%H%M%S)
 if [ -d "$BACKUP" ]
 then
-    mkdir -p "$BACKUP"-OLD/"$DATE"
+    rm -rf "$BACKUP"-OLD
+    mkdir -p "$BACKUP"-OLD
     install_if_not rsync
-    rsync -Aaxz "$BACKUP"/ "$BACKUP"-OLD/"$DATE"
+    rsync -Aaxz "$BACKUP"/ "$BACKUP"-OLD
```
Comment (szaimen) [inline nextcloud_update.sh:740]: What about renaming the backup folder here instead of rsyncing the content?

Comment (szaimen) [reply nextcloud_update.sh]: done

**Comment by enoch85** [comment]
Thanks!

I think a better implementation would be to delete `older than X days`.

This  specific commit would also delete all old backups for current users, I'm not sure we want that.

**Comment by enoch85** [comment]
Maybe something like: `find "$BACKUP"-OLD/*" -mtime +90 -exec rm {} \;`

Another option would be to move all the data to the /mnt/ncdata instead... That's the original location but is changed during the script for some reason I can't remember: https://github.com/nextcloud/vm/blob/b6edbac199cd2a3c5ed640b460459b98dfda9fe1/nextcloud_update.sh#L703

**Comment by szaimen** [comment]
@enoch85 as you wanna do it pretty differently than my proposal, I suggest that you open a new PR and implement there how you would do it and then I can have a look if I am okay with this. What do you think?

**Pull request closed without merging**